### DISCUSSION TITLE
Does `init_graphql_request` fire for GET requests?

### DISCUSSION THREAD

**Discussion by alancwoo** [wp-graphql/wp-graphql#3140] [Q&A]
I have an action running on `init_graphql_request` which seems to work with `POST` requests but not `GET` requests via `?query=...`

I'm wondering if I'm testing this incorrectly or if somehow this doesn't fire on `GET`?

**Comment by justlevine** [comment]
AFAIK `init_graphql_request` should fire on the _initial_ `GET` request, but not on subsequent requests, since those are cached and not actually hitting the server.
